### PR TITLE
Migrate speech to text example app to swift package manager & fix compile errors

### PR DESCRIPTION
**Pull request by Jake-Prickett**
Motivation:

After reading that [SwiftNIO was dropping support for CocoaPods](https://forums.swift.org/t/swiftnio-is-dropping-support-for-cocoapods/56840) I thought it'd be fun to revisit the example App and refresh it. 

I also believe this PR resolves https://github.com/grpc/grpc-swift/issues/1393

Modifications:

- Removed Cocoapods from the Speech to Text Example App
- Added missing `resource.proto` to generation step (Must have been added recently)
- Added the required Swift Packages to the project

### CODE REVIEW

File: Examples/Google/SpeechToText/Sources/SpeechService.swift [hunk 1]
```diff
@@ -16,6 +16,7 @@
 
 import GRPC
 import Logging
+import Foundation
```
Comment (glbrntt) [inline Examples/Google/SpeechToText/Sources/SpeechService.swift:19]: The formatter doesn't like the order of these imports, I don't think we need `Foundation` here so let's just remove it.

```suggestion
```

**Review by glbrntt — COMMENTED** [review]
Thanks @Jake-Prickett, just one suggestion left inline.

File: Examples/Google/SpeechToText/Sources/SpeechService.swift (hunk 1 above)
Comment (Jake-Prickett) [reply Examples/Google/SpeechToText/Sources/SpeechService.swift]: When I remove the import, the compiler has trouble finding `Data` for the `stream(_ data: Data, ..)` method below. Not 100% certain, my hunch would be that when removing Cocoapods a project setting changed that allowed it to not be required. 

I've corrected the order of the imports 👍

Comment (glbrntt) [reply Examples/Google/SpeechToText/Sources/SpeechService.swift]: Ah, great, thanks!

**Review by glbrntt — APPROVED** [review]

**Pull request merged**